app/views/posts/index.html.erb

```diff
@@ -14,26 +14,38 @@
 </div>
 
 <section>
- <%@user.most_recent_posts.each_with_index do |post, index| %>
-   <%=link_to user_post_path(@user, post.id) do %>
+ <%@user.most_recent_posts.each do |post| %>
     <div class=" border border-dark border-3 mx-auto w-100 p-3 mb-3 text-dark">
-        <h3> Post #<%= index + 1 %></h3>
+        <h3> <%= link_to post.title, "/users/#{@user.id}/posts/#{post.id}" %> </h3>
         <p> <%= post.text %></p>
-    <div class="d-flex justify-content-end">
+    <div class="d-flex justify-content-between">
+       <div>
+          <%= form_with url: "/likes/#{post.id}" do |form| %>
+          <%= form.submit "Like", class: "btn btn-primary" %>
+          <% end %>
+       </div>
```

Comment:
- [ ] [optional] @brhanuhailu great work implementing the changes requested, however, I would suggest you move the like button to the post-show page as previously requested, its industry convention thou not a requirement to have comments and likes on same page for better User Experience